test/controllers/ViewTransfersControllerSpec.scala

```diff
@@ -30,13 +30,66 @@ class ViewTransfersControllerSpec extends FrontEndTestHelper {
   val checkPageForStaticElements = new CheckPageForStaticElements
 
   private val formatter = DateTimeFormatter.ofPattern("dd/MM/yyyy HH:mm:ss")
+  private val defaultCurrentStatus = CurrentStatus(None, None, None, None, None, None)
+  private val standardType = "standard"
+  private val judgmentType = "judgment"
 
   implicit val ec: ExecutionContext = ExecutionContext.global
 
+  forAll(ConsignmentStatusesOptions.expectedStandardStatesAndStatuses) { (expectedTransferState, currentStatus, actionUrl, transferState, actionText) =>
+    {
+      s"ViewTransfersController for '$standardType' consignments" should {
+        s"render the '$expectedTransferState' action for the given 'consignment status'" in {
+          setConsignmentTypeResponse(wiremockServer, "standard")
+          val consignment: List[Consignments.Edges] = setConsignmentViewTransfersResponse(wiremockServer, standardType, customStatusValues = List(currentStatus))
+
```

Comment:
unnecessary line break here. There's a few more in the other test cases too i.e. line 119, 143, 168